src/qt/peertablemodel.cpp

```diff
@@ -33,6 +43,97 @@ PeerTableModel::~PeerTableModel()
     // Intentionally left empty
 }
 
+void PeerTableModel::DrawIcons()
+{
+    static constexpr auto SIZE = 32;
+    static constexpr auto ARROW_HEIGHT = SIZE * 2 / 3;
+    QImage icon_in(SIZE, SIZE, QImage::Format_Alpha8);
+    icon_in.fill(Qt::transparent);
+    QImage icon_out(icon_in);
+    QPainter icon_in_painter(&icon_in);
+    QPainter icon_out_painter(&icon_out);
+
+    // Arrow
+    auto DrawArrow = [](const int x, QPainter& icon_painter) {
+        icon_painter.setBrush(Qt::SolidPattern);
+        QPoint shape[] = {
+            {x, ARROW_HEIGHT / 2},
+            {(SIZE-1) - x,  0},
+            {(SIZE-1) - x, ARROW_HEIGHT-1},
+        };
+        icon_painter.drawConvexPolygon(shape, 3);
+    };
+    DrawArrow(0, icon_in_painter);
+    DrawArrow(SIZE-1, icon_out_painter);
+
+    {
+        const QString label_in  = tr("IN" , "Label on inbound connection icon");
```

Comment:
We do not use disambiguation strings anymore. This would need to be made into a translator comment.

diff for Line 70 and 71:

``` diff

diff --git a/src/qt/peertablemodel.cpp b/src/qt/peertablemodel.cpp
index 58998781c..8f65f3e75 100644
--- a/src/qt/peertablemodel.cpp
+++ b/src/qt/peertablemodel.cpp
@@ -67,8 +67,10 @@ void PeerTableModel::DrawIcons()
     DrawArrow(SIZE-1, icon_out_painter);
 
     {
-        const QString label_in  = tr("IN" , "Label on inbound connection icon");
-        const QString label_out = tr("OUT", "Label on outbound connection icon");
+        //: Label on inbound connection icon
+        const QString label_in  = tr("IN");
+        //: Label on outbound connection icon
+        const QString label_out = tr("OUT");
         QImage scratch(SIZE, SIZE, QImage::Format_Alpha8);
         QPainter scratch_painter(&scratch);
         QFont font;  // NOTE: Application default font

```

Reply:
k